website/pages/docs/job-specification/service.mdx

```diff
@@ -154,9 +154,9 @@ Connect][connect] integration.
 
   - `host` - Use the host IP and port.
 
-- `task` `(string: "")` - Specifies the name of the Nomad Task associated with
+- `task` `(string: "")` - Specifies the name of the Nomad task associated with
   this service definition. Only available on group services. Must be set if this
-  service definition represents a Consul Connect Native service and there is more
+  service definition represents a Consul Connect native service and there is more
```

Comment:
The content of https://www.consul.io/docs/connect/native suggests that the canonical representation should be "Connect-native".

```suggestion
  service definition represents a Consul Connect-native service and there is more
```